docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -163,7 +163,7 @@ For more information, see the https://github.com/zchee/docker-machine-driver-xhy
 To use Minishift with Hyper-V:
 
 . Install https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[Hyper-V].
-. Add a https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch[Virtual Switch].
+. Add an https://msdn.microsoft.com/en-us/virtualization/hyperv_on_windows/quick_start/walkthrough_virtual_switch[External Virtual Switch].
```

Comment:
IMO no need to make the "external" word in bold, the link format makes it visible anyway. Also, since it's not a part of the proper noun "Virtual Switch" we shouldn't make the "E" capitalized.